lib/MDTable.js

```diff
@@ -0,0 +1,69 @@
+import AsciTable from 'ascii-table';
+
+class MDTable extends AsciTable {
+    /**
+     * Create a CSV string from the table data
+     * @returns {string}
+     */
+    toCSV() {
+        return [this.getHeading(), ...this.getRows()].join('\n');
+    }
+
+    /**
+     * Create a JSON string from the table data
+     * @returns {string}
+     */
+    toJSON() {
+        return JSON.stringify(super.toJSON());
+    }
+
+    /**
+     * Create a MD string from the table data
+     * @returns {string}
+     */
+    toString() {
+        // Ignore modifying the first |---| for titled tables
+        let idx = this.getTitle() ? -2 : -1;
+        const ascii = super.toString()
+            .replace(/-\|/g, () => {
+                /* we replace "----|" with "---:|" to align the data to the right in MD */
+                idx++;
+
+                if (idx < 0 || this.leftAlignedCols.includes(idx)) {
+                    return '-|';
+                }
+
+                return ':|';
+            });
+
+        // strip the top and the bottom row (----) to make an MD table
+        const md = ascii.split('\n').slice(1, -1).join('\n');
+        return md;
```

Comment:
The tables are surrounded by `------` border on the top and the bottom `------`
I was always removing them when I was copying as it didn't render correctly in MD